buildSrc/src/main/java/org/opensearch/gradle/test/DistroTestPlugin.java

```diff
@@ -77,9 +77,9 @@
 import java.util.stream.Stream;
 
 public class DistroTestPlugin implements Plugin<Project> {
-    private static final String SYSTEM_JDK_VERSION = "21.0.6+7";
+    private static final String SYSTEM_JDK_VERSION = "23.0.2+7";
     private static final String SYSTEM_JDK_VENDOR = "adoptium";
-    private static final String GRADLE_JDK_VERSION = "21.0.6+7";
+    private static final String GRADLE_JDK_VERSION = "23.0.2+7";
     private static final String GRADLE_JDK_VENDOR = "adoptium";
```

Comment:
On main we have switched back to JDK21 and JDK23 has been deprecated.
Seems like JDK24 has just released yesterday.

Thanks.

Reply:
> Seems like JDK24 has just released yesterday.

No worries,  I will be testing with JDK-24 asap, thank you